### ISSUE TITLE
UCSD 6 Install fails 

### ISSUE THREAD

**Issue by noMoreCLI** [CiscoUKIDCDev/ucsd-spark-plugin#10]
Installation on UCS Director 6.0 fails with
Error: No Module properties file found.

Spark Plugin created on a MacOS X (10.12) with Java 1.8.0_101/Ant 1.9.7
<img width="485" alt="screen shot 2016-09-28 at 08 49 15" src="https://cloud.githubusercontent.com/assets/22365813/18903319/b73e111a-8558-11e6-90ec-fddc3478eb27.png">

**Comment by mattsday** [comment]
I just tried this and it works for me when compiling via Eclipse.

Can you try the attached (compiled via ant on OS X 10.12 and Java 1.8.0_92-b14 / ant 1.9.7)?
[Spark-plugin.zip](https://github.com/CiscoUKIDCDev/ucsd-spark-plugin/files/497764/Spark-plugin.zip)

```
-----BEGIN PGP SIGNATURE-----
Version: GnuPG/MacGPG2 v2.0
Comment: GPGTools - https://gpgtools.org

iQEcBAABCgAGBQJX651+AAoJEAXmeSUDjMcJ7p0H/jHDm8S1UoRWxLO4pnpZoEKZ
MAMqtK1+xYil6PopZvDLmUEwmrObeA/yyrzOifYyOzKOJLsujqsDoyTNN2eMlb3G
UsBIzLJ1CLx271wyBhBX2RuXZmsafQDXxJxCPXifa19aWAh1Eehr42rVhaWCIbmI
fDUthG/MKWuVJyYKqsSpZOjQPTuib1d5NS87ErZWByfn/aJeafoIYTJbxcgPNluU
1hSP8LG/vqJoqvFkgcmDjLOLTP6OJNSCnb6+hJ8ZqCZeccaySpPDfS2Yub77HKgN
mMO0CWzEaUjjxNTcxiLUZuE79vBUyAvATVAn+12MRzV9FgKPXb6rAQo793QynmQ=
=okWL
-----END PGP SIGNATURE-----
```

**Comment by mattsday** [comment]
Fixed in e83fcac
